src/domain/event/eventNavigation/EventNavigation.tsx

```diff
@@ -22,25 +24,47 @@ interface Props {
 
 const EventNavigation: React.FC<Props> = ({ items }) => {
   const { t } = useTranslation();
+  const history = useHistory();
+  const location = useLocation();
   const { theme } = useTheme();
-  const [activeTab, setActiveTab] = React.useState(0);
+  const searchParams = new URLSearchParams(location.search);
 
+  const defaultTab: number = React.useMemo(() => {
+    const tab = Number(searchParams.get(URL_PARAMS.TAB));
+
+    if (Number.isInteger(tab)) {
+      return tab < items.length && !items[tab].disabled ? tab : 0;
+    }
+    return 0;
+  }, [items, searchParams]);
+
+  const [activeTab, setActiveTab] = React.useState(defaultTab || 0);
+
+  const handleActiveTabChange = (index: number) => {
+    const search = location.search;
+    const searchParams = new URLSearchParams(search);
+
+    setActiveTab(index);
+
+    searchParams.set(URL_PARAMS.TAB, index.toString());
+    history.push({ ...location, search: searchParams.toString() });
```

Comment:
spreading location might be redundant here

Reply:
Would 
```
const handleActiveTabChange = (index: number) => {
    const search = location.search;
    const searchParams = new URLSearchParams(search);

    setActiveTab(index);

    searchParams.set(URL_PARAMS.TAB, index.toString());

    location.search = searchParams.toString();
    history.push(location);
  };
``` 
be clearer? Or did I miss your point?

Reply:
Maybe you missed my point, I wasn't clear enough. I meant that passing the whole location object might be redundant. Would it work correctly with only the search key?

`history.push({ search: searchParams.toString() });`

Reply:
Ah, I can do that. Great! 
Changed

Reply:
It caught my eye and I tested it locally when I was reviewing. Seemed to work correctly :)

Reply:
I should have tested that myself :D

Reply:
I suggest that you do now :D

Reply:
I did and it worked

Reply:
👍